wandb/sdk/launch/utils.py

```diff
@@ -690,7 +690,7 @@ def docker_image_exists(docker_image: str, should_raise: bool = False) -> bool:
 
 def pull_docker_image(docker_image: str) -> None:
```

Comment:
we should audit the use of `pull_docker_image` in our code base, it doing duplicate work with `docker_image_exists` feels wrong since it means this PR: https://github.com/wandb/wandb/pull/5531 didn't actually do what you intended.

Reply:
Dusting this off. This is the only usage actually.